web/src/lib/utils/asset-utils.ts

```diff
@@ -173,3 +173,15 @@ export function getAssetRatio(asset: AssetResponseDto) {
   }
   return { width, height };
 }
+
+// list of supported image extensions from https://developer.mozilla.org/en-US/docs/Web/Media/Formats/Image_types
+const supportedImageExtensions = ['apng', 'avif', 'gif', 'jpg', 'jpeg', 'jfif', 'pjpeg', 'pjp', 'png', 'svg', 'webp'];
```

Comment:
Remove SVG here.

Reply:
I also turned the array into a Set to improve performance while checking if an extension is supported